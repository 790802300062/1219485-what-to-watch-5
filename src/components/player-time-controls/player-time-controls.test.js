import React from "react";
import renderer from "react-test-renderer";
import PlayerTimeControls from "./player-time-controls";

describe(`PlayerTimeControls render`, () => {
  it(`renders PlayerTimeControls component with runtime 180 and currentTimeInSeconds 100`, () => {
    const tree = renderer
      .create(
          <PlayerTimeControls
            runtime={180}
            currentTimeInSeconds={100}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`renders PlayerTimeControls component with runtime 180 and currentTimeInSeconds 1000`, () => {
    const tree = renderer
      .create(
          <PlayerTimeControls
            runtime={180}
            currentTimeInSeconds={1000}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
