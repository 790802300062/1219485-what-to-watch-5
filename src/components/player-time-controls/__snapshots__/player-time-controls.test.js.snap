// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerTimeControls render renders PlayerTimeControls component with runtime 180 and currentTimeInSeconds 100 1`] = `
<div
  className="player__controls-row"
>
  <div
    className="player__time"
  >
    <progress
      className="player__progress"
      max="100"
      value={1}
    />
    <div
      className="player__toggler"
      style={
        Object {
          "left": "1%",
        }
      }
    >
      Toggler
    </div>
  </div>
  <div
    className="player__time-value"
  >
    10:20:00
  </div>
</div>
`;

exports[`PlayerTimeControls render renders PlayerTimeControls component with runtime 180 and currentTimeInSeconds 1000 1`] = `
<div
  className="player__controls-row"
>
  <div
    className="player__time"
  >
    <progress
      className="player__progress"
      max="100"
      value={9}
    />
    <div
      className="player__toggler"
      style={
        Object {
          "left": "9%",
        }
      }
    >
      Toggler
    </div>
  </div>
  <div
    className="player__time-value"
  >
    19:20:00
  </div>
</div>
`;
